packages/hasher/src/yarnWorkspaces.ts

```diff
@@ -0,0 +1,119 @@
+import * as path from "path";
+import * as fg from "fast-glob";
+
+import * as findWorkspaceRoot from "find-yarn-workspace-root";
+
+type PackageJsonWorkspaces = {
```

Comment:
is this type copied from the TypeScript repo? Are we sure that this covers all ways that workspaces could be express (eg. maybe it supports glob)

Reply:
https://github.com/yarnpkg/yarn/blob/6e5b0ea0dc15bef7d2abd67a46adbe7efe533404/src/types.js#L164